README.md

```diff
@@ -25,39 +42,32 @@ rdvue --help
 ### Usage
 
 ```
-RDVUE CLI [Node: 12.13. 1, CLI: 0.0.6]
-
 rdvue [command]
 
 Commands:
   rdvue <action> <feature> <feature name> [options]
 
-
-rdvue [actions]
-
 Actions:
-  generate  g    Creates new feature
-
-
-rdvue [features]
+  generate  Creates new feature
+  g         Creates new feature
 
 Features:
   project       Generates a new Vue.js Typescript Single Page Application(SPA)
-  component     Generates shell for a basic component with all its base boilerplate code
-  page          Generates shell for a basic page with all its base boilerplate code
-  service       Generates shell for a basic service with all its base boilerplate code
-  model         Generates shell for a basic model/interface
-
+  component     Generates a new component directory with the following main files to build and test: <component-name>.vue, <component-name>.ts, <component-name>.scss, <component-name>.spec.ts
+  page          Generates a new page directory with the following main files to build and test: <page-name>.vue, <page-name>.ts, <page-name>.scss, <page-name>.spec.ts
+  service       Generates a new service file within the services directory to integrate methods that should interact with external apis: <service-name>.ts
+  model         Generates a new model/interface within the entities directory: <interface-name>.ts
 
 Options:
   --help        Show help                                                   [boolean]
+  -h            Show help                                                   [boolean]
 ```
 
 ## Contributing
 
-RD VUE CLI is a part of the FP Frontend Practice area, where contributions are welcomed, You can help us by fixing bugs, reporting bugs or improving documentation.
+RD VUE CLI is a part of the FP Frontend Practice area at RealDecoy, where contributions are welcomed, You can help us by fixing bugs, reporting bugs or improving documentation.
```

Comment:
for consistency of the name **rdvue** lets change this sentence to read "The rdvue CLI is..."